tests/usm/usm_atomic_access_atomic64_fp64.cpp

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification to atomic access for USM allocations that underlying
+//  type size equal than 64 byte.
```

Comment:
```suggestion
//  type size equal to 64 byte.
```
Is this a typo?
If so, please, fix in tests/usm/usm_atomic_access_atomic64.cpp as well.

Reply:
The comments have been reworded for both tests.